spec/requests/ramen_shops_spec.rb

```diff
@@ -26,10 +26,31 @@
     it_behaves_like 'when not logged in'
     it_behaves_like 'as a non-admin'
 
-    it 'returns new form when logged in as an admin' do
-      log_in_as admin
-      do_request
-      expect(response.body).to include '<h1>店舗登録</h1>'
+    context 'with admin' do
+      let(:ramen_shop) { controller.instance_variable_get(:@ramen_shop) }
+
+      before { log_in_as admin }
+
+      context 'without params' do
+        it 'initializes a new RamenShop with empty attributes' do
+          get new_ramen_shop_path
+          expect(ramen_shop.name).to be_nil
+          expect(ramen_shop.address).to be_nil
+        end
+      end
```

Comment:
[FYI]
変更は不要ですが参考までに

let(:ramen_shop) { controller.instance_variable_get(:@ramen_shop) }
の部分を上部に定義するとテストを読む際にめが上下するため、可読性が落ちる且つ、全てのitで利用している訳ではないので、変数で定義するのではなく、冗長ではありますが、利用するitの中で取得する方が、明白なテストになりそうです！

```suggestion
      before { log_in_as admin }

      context 'without params' do
        it 'initializes a new RamenShop with empty attributes' do
          get new_ramen_shop_path
          ramen_shop = controller.instance_variable_get(:@ramen_shop)
          expect(ramen_shop.name).to be_nil
          expect(ramen_shop.address).to be_nil
        end
      end
```

Reply:
ありがとうございます！確かに使っていないテストケースもあるため、それぞれのテスト内で定義するよう修正いたします。